lib/house.rb

```diff
@@ -0,0 +1,47 @@
+class House
+
+  attr_reader :price,
```

Comment:
Since you have a price method below, you shouldn't include an attr_reader for price.